src/js/containers/Submission.jsx

```diff
@@ -10,6 +10,7 @@ import SubmissionPageInfo from '../components/SubmissionPageInfo.jsx'
 import UploadForm from './UploadForm.jsx'
 import ErrorWarning from '../components/ErrorWarning.jsx'
 import EditsContainer from './Edits.jsx'
+import Receipt from './Receipt.jsx'
```

Comment:
It's picky, but I think we should rename this `SubmissionReceipt`. At least it better matches `SubmissionPageInfo` and that it falls on the submission page.

_Which brings up the directories again._ 😉